packages/reactivity/src/computed.ts

```diff
@@ -59,6 +60,10 @@ export class ComputedRefImpl<T> {
     if (self._dirty || !self._cacheable) {
       self._dirty = false
       self._value = self.effect.run()!
+    } else if (__DEV__ && self._dirty && self._cacheable) {
```

Comment:
I'm not sure I'm following. When will this warning be shown?

If `self._dirty` is `true` then it'll go into the `if` branch, so I don't see how it can ever go into this `else if`.

Reply:
Oh, it's my mistake. I have already fixed it. Thx

Reply:
But now it just logs a warning whenever a `computed` gets accessed during rendering:

[SFC Playground example](https://deploy-preview-9081--vue-sfc-playground.netlify.app/#eNp9UT1PwzAQ/SsnL/1QCRLdqrYSoA4wAAJGL1F6CSmObexziBT5v2M7JDBU3Xzvw++drme3WmetQ7ZhW1uYWhNYJKf3XNaNVoagB4PlCgrVaEd4BA+lUQ3MgmnGJZeFkpZAugZ2UTm/WYyY/XK5wQCP3vl8Abt91GZtLhzCcglRvr0eokNoGAgbLXLCMAH0ffrae7jax+H3T++ja1KyFSMbQsu6yk5WybBNH92cxehaoHnWVIdSnG0gMZHLhVDfjwkj43A14sUHFp9n8JPtIsbZi0GLpkXOJo5yUyEN9OHtCbvwnshGHZ0I6gvkK1olXOw4yO6cPIba/3Sp7UO6SS2rd3voCKUdl4pFo9InPWfhOvcXVv+ru87WycelZ/4HCUWuzg==)

There shouldn't be a warning, that's just normal usage.

The unit tests are failing for your branch due to the new warnings.

Please make sure you have checked your changes thoroughly before pushing them to a PR. In particular, confirm that they do what you're expecting locally and that all of the tests still pass. It seems that you're making changes without actually running the code to check that it works.